arkouda/pdarrayIO.py

```diff
@@ -581,3 +588,146 @@ def save_all(columns : Union[Mapping[str,pdarray],List[pdarray]], prefix_path :
             first_iter = False
         else:
             arr.save(prefix_path=prefix_path, dataset=name, file_format=file_format, mode='append')
+
+
+def import_data(read_path: str, write_file: str = None,
+                return_obj: bool = False, index: bool = False):
+    """
+    Import data from Pandas to Arkouda readable format
```

Comment:
Maybe a longer explanation here? "import data" makes me think we are importing data into arkouda. Are we just re-writing it in a different format? Which pandas formats are allowed? Is it only from a file or can I pass it a pandas object directly?

Question: Let's say I have an arkouda dataframe, akdf, with special column types, like bitvector, ip address, etc. I transfer this to a local pandas object by akdf.to_pandas(), then write it out to a parquet or hdf5 file using pandas. If I use this "import_data" function, will I get back what I started with? If not, that would be important to explain here.

Reply:
I updated the docstring to hopefully help. You have the option to return the object or save a file. Import/Export is intended to be done using a saved file. The DataFrame d`__init__` already accepts and pandas dataframe object to build from. That is actually how objects are built, so to answer your scenario, yes you should get back what you started with.